lib/lib_phpQuery.php

```diff
@@ -1033,7 +1033,22 @@ public function __construct($paramList, $code, $param1 = null, $param2 = null,
 			$param3 = null) {
 		$params = func_get_args();
 		$params = array_slice($params, 2);
-		$this->callback = create_function($paramList, $code);
+
+		$this->callback = function(&...$args) use ($paramList, $code) {
+			$paramListArray = explode(',', $paramList);
```

Comment:
Is this function copied from somewhere else? If yes, please add a reference as a comment. If not, fine :-)

Reply:
@Alkarex, this function is not copied, totally self-written.

Reply:
Ok 👍